GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTFLLUploader.m

```diff
@@ -328,4 +353,23 @@ - (void)appWillTerminate:(GDTCORApplication *)application {
   });
 }
 
+#pragma mark - NSURLSessionDelegate
+
+- (void)URLSession:(NSURLSession *)session
+                          task:(NSURLSessionTask *)task
+    willPerformHTTPRedirection:(NSHTTPURLResponse *)response
+                    newRequest:(NSURLRequest *)request
+             completionHandler:(void (^)(NSURLRequest *_Nullable))completionHandler {
+  if (!completionHandler) {
```

Comment:
nit: The parameter is not marked as an optional, not sure if we should validate it anyways, though it doesn't harm.

Reply:
Only newer sanitizer versions catch this and help you solve it. On older versions of Xcode, it'd just crash without much help. Since invoking a nil block is a hard crash, I'd rather keep the check in.

Reply:
Why the block would be nil?

Reply:
Why wouldn't it be? Until recently, the annotations were more suggestions than hard rules. I'm not in control of what calls or passes data into this method, therefore, it's prudent to validate input.